### ISSUE TITLE
Blocking after the last stage

### ISSUE THREAD

**Issue by Ankou-B** [SoMainline/linux-apple-resources#18]
How do I enter commands in the linux shell when I'm at this stage?
![IMG_4751](https://github.com/SoMainline/linux-apple-resources/assets/124830325/b044a26f-7ed5-4f57-a039-46f30745baa4)

**Comment by HODAKdev** [comment]
Hi, its iphone 7?

**Comment by Ankou-B** [comment]
No it is an iphone 5s

> Hi, its iphone 7?

**Comment by HODAKdev** [comment]
> No it is an iphone 5s
> 
> > Hi, its iphone 7?

How did you manage to run the Linux kernel on an unsupported iPhone?

**Comment by Ankou-B** [comment]
it's supported it's A7

**Comment by Ankou-B** [comment]
but i don't know how to enter command at this point, how interact with the kernel

**Comment by HODAKdev** [comment]
Here
https://wiki.postmarketos.org/wiki/Inspecting_the_initramfs

**Comment by Ankou-B** [comment]
it doesnet work, when i want to connect to
telnet 172.16.42.1
im bloqued on "Trying 172.16.42.1..."
and when i use dmesg
i got
"65678.961794] usb 1-1.2: new high-speed USB device number 61 using ehci-pci
[65679.070859] usb 1-1.2: New USB device found, idVendor=05ac, idProduct=4141, bcdDevice= 0.00
[65679.070867] usb 1-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[65679.070869] usb 1-1.2: Product: pongoOS USB Device
[65679.070870] usb 1-1.2: Manufacturer: checkra1n team
[65679.070871] usb 1-1.2: SerialNumber: pongoOS / checkra1n 2.5.1-5cb3b4fb
[65680.865068] usb 1-1.2: USB disconnect, device number 61
[65686.701635] usb 1-1.2: new high-speed USB device number 62 using ehci-pci
[65691.901403] usb 1-1.2: device descriptor read/64, error -110
[65707.517024] usb 1-1.2: device descriptor read/64, error -110
[65707.705010] usb 1-1.2: new high-speed USB device number 63 using ehci-pci
[65712.892814] usb 1-1.2: device descriptor read/64, error -110
[65728.508398] usb 1-1.2: device descriptor read/64, error -110
[65728.616519] usb 1-1-port2: attempt power cycle
[65729.220371] usb 1-1.2: new high-speed USB device number 64 using ehci-pci
[65739.900105] usb 1-1.2: device not accepting address 64, error -110
[65739.980106] usb 1-1.2: new high-speed USB device number 65 using ehci-pci
[65750.651756] usb 1-1.2: device not accepting address 65, error -110
[65750.651987] usb 1-1-port2: unable to enumerate USB device
[66311.234555] usb 1-1.2: new high-speed USB device number 66 using ehci-pci
[66316.526221] usb 1-1.2: device descriptor read/64, error -110
[66332.137623] usb 1-1.2: device descriptor read/64, error -110
[66332.325615] usb 1-1.2: new high-speed USB device number 67 using ehci-pci
[66337.513401] usb 1-1.2: device descriptor read/64, error -110
[66361.904572] usb 1-1.2: new high-speed USB device number 68 using ehci-pci
[66367.208299] usb 1-1.2: device descriptor read/64, error -110"

I don't know what to do

**Comment by Ankou-B** [comment]
and my usb cable work fine

**Comment by intervade** [comment]
you can try to follow the iphone 6 postmarketos page. you have to go through their ramdisk process and enable netboot and you should be able to ssh in afterward.

**Comment by Ankou-B** [comment]
no, when i do that, postmarketOS say "unable to enable netboot"
